packages/expo-cli/src/commands/eas-build/build/action.ts

```diff
@@ -205,6 +211,11 @@ async function startBuildAsync<T extends Platform>(
     const job = await builder.prepareJobAsync(archiveUrl);
     log(`Starting ${platformDisplayNames[job.platform]} build`);
 
+    if (process.env.EAS_OUTPUT_JOB_JSON === '1') {
+      process.stdout.write(`>>> EAS_JOB_START\n${JSON.stringify(job)}\n>>> EAS_JOB_END`);
+      process.exit(0);
```

Comment:
Are you sure that everything from the stdout will be printed?

Reply:
I think I don't understand the question.

Reply:
stdout is buffered, so it's possible that calling process exit will not display everything

Reply:
from some searching, it seems that when piping, `stdout.write` is blocking so it should print everything. though in normal usage it's probably very unlikely that the buffer is full so probably not something we need to worry about for now.

probably in future when we add user facing UI for this, we can do some refactor so that we wouldn't have to call `process.exit`, which should address the issue.